app.js

```diff
@@ -157,7 +157,7 @@ class MyApp extends Homey.App {
     console.log(`Webhook address: ${webhook}`);
     this.homey.settings.set('webhook', webhook);
     let retryCount = 10;
-    while (retryCount > 0) {
+    while (retryCount > 0 && webhookId != null) {
```

Comment:
I had to add this to be able to run locally. Not sure why, but I get these warnings:
Warning: Trying to access `Homey.env.WEBHOOK_ID` but the environment variable has not been set!
Warning: Trying to access `Homey.env.WEBHOOK_SECRET` but the environment variable has not been set!
and then it explodes because of this.

Reply:
You have to add a file called env.json, it should look like this:
```
{
  "WEBHOOK_ID": "replace this text with webhook id created on https://tools.developer.homey.app/webhooks",
  "WEBHOOK_SECRET": "replace this text with webhook secret created on https://tools.developer.homey.app/webhooks"
}
```